lib/gitarro/opt_parser.rb

```diff
@@ -182,6 +190,7 @@ def defaults_to_text
     @options[:file_type] = '.changes' if @options[:changelog_test]
     @options[:description] = desc if @options[:description].nil?
     @options[:file_type] = 'notype' if @options[:file_type].nil?
+    @options[:cachehttp] = '/tmp' if @options[:cachehttp].nil?
```

Comment:
Here it should be:

```@options[:cachehttp] = '/tmp/gitarro' if @options[:cachehttp].nil?```

Reply:
gitarro dir is created by the active_support cache , so it is good to have only `tmp` here

Reply:
What do you mean? The module creates it and you CAN'T specify another value?

Reply:
yes WE CAN :us: